frontend/.idea/codeStyles/Project.xml

```diff
@@ -19,6 +19,7 @@
       <option name="SPACES_WITHIN_IMPORTS" value="true" />
     </TypeScriptCodeStyleSettings>
     <codeStyleSettings language="JavaScript">
+      <option name="RIGHT_MARGIN" value="100" />
       <option name="SOFT_MARGINS" value="100" />
     </codeStyleSettings>
     <codeStyleSettings language="TypeScript">
```

Comment:
please remove the .idea folder from git.

Reply:
I thought it's in gitignore